docs/index.md

```diff
@@ -6,20 +6,25 @@ Equinix Labs offers workshops, proof of concepts, and tools for exploring and bo
 
 ## About the workshop
 
-The goals of this workshop are:
+Welcome to the workshop on provisioning Equinix infrastructure using Pulumi. Throughout this workshop, we will explore the capabilities of Pulumi, leveraging its versatility through multi-language and multi-cloud support. Our demonstration will focus on utilizing a Pulumi template written in Python to establish a private connection from a new Equinix Fabric Cloud Router to Google Cloud Platform (GCP). We will utilize the Pulumi providers for Equinix and GCP.
+
+It's important to note that currently, there's a limitation with the GCP provider, as it does not include the option to update pre-created BGP configuration, necessitating a manual step to add the BGP peer ASN. Nevertheless, this functionality is achievable using the GCP API and its Python SDK. Therefore, in this example, we will explore extending the functionality of Pulumi providers with programming logic, achieving end-to-end automation.
```

Comment:
> It's important to note that currently, there's a limitation with the GCP provider

In this workshop, we'll seamlessly take advantage of a cloud provider SDK to use capabilities not present in the corresponding Terraform or Pulumi providers offered by the cloud provider.